bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -194,8 +194,66 @@ class PartialRowsData(object):
 
     def __init__(self, response_iterator):
         self._response_iterator = response_iterator
+        self.generator = YieldRowsData(response_iterator)
+
         # Fully-processed rows, keyed by `row_key`
         self._rows = {}
+
+    def __eq__(self, other):
+        if not isinstance(other, self.__class__):
+            return False
```

Comment:
Please `return NotImplemented` here. See, e.g. `Entity.__eq__` in datastore

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/0b6df3d8ba76a9ed917ff94357a7fa7eb16a7582/datastore/google/cloud/datastore/entity.py#L154-L155